themes/classic/_dev/js/components/form.js

```diff
@@ -0,0 +1,32 @@
+import $ from 'jquery';
+
+export default class Form {
+  init(){
+    this.parentFocus();
+    this.togglePasswordVisibility();
+  }
+
+  parentFocus() {
+    $('.js-child-focus').focus(function () {
+      $(this).closest('.js-parent-focus').addClass('focus');
+    });
+    $('.js-child-focus').focusout(function () {
+      $(this).closest('.js-parent-focus').removeClass('focus');
+    });
+  }
+
+  togglePasswordVisibility() {
+    $('button[data-action="show-password"]').on('click', function () {
```

Comment:
As you are working on it, would you like to be the first following our next conventions, following Airbnb?

https://github.com/airbnb/javascript#arrows--use-them

```js
on('click', () => {
```